delphi-checks/src/main/resources/org/sonar/l10n/delphi/rules/community-delphi/IndexLastListElement.json

```diff
@@ -1,5 +1,5 @@
 {
-  "title": "The last element in a `TList` descendant should be retrieved with the `Last` method.",
+  "title": "The last element in a `TList` descendant should be retrieved with the `Last` method",
```

Comment:
While we're in here, may as well swap these backticks to single-quotes for consistency with the other rule titles.